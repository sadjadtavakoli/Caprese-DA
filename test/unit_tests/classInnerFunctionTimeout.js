class TextInlines {

    constructor(pdfDocument) {
        this.pdfDocument = pdfDocument;
    }
    buildInlines() {

        setTimeout(myFunction, 500);
        setTimeout(myFunction, 100);
        setTimeout(() => {

        }, 100);

        function myFunction() {
            //  do nothing
        }


    }
}


let textInlines = new TextInlines("pdfDocument");
textInlines.buildInlines()
