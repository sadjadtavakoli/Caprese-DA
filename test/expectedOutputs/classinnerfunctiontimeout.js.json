{"TextInlines-test/unit_tests/classinnerfunctiontimeout.js-1-20":{"impacted":["test/unit_tests/classinnerfunctiontimeout.js"]},"test/unit_tests/classinnerfunctiontimeout.js":{"impacted":["TextInlines-test/unit_tests/classinnerfunctiontimeout.js-1-20"]},"buildInlines-test/unit_tests/classinnerfunctiontimeout.js-6-19":{"impacted":["test/unit_tests/classinnerfunctiontimeout.js"]}}